crates/cli/src/lib.rs

```diff
@@ -180,7 +180,7 @@ pub fn build_app() -> Command {
         PossibleValuesParser::new(Target::iter().map(Into::<&'static str>::into));
 
     Command::new("roc")
-        .version(concatcp!(VERSION, "\n"))
+        .version(VERSION)
```

Comment:
I removed this newline because I don't think it's needed, but I could be wrong :thinking: